_data/settings/components/table.yml

```diff
@@ -35,6 +40,11 @@ contents:
     var: $theme-table-sorted-stripe-background-color
     default: '"blue-cool-10v"'
     type: color
+  - name: Sticky header top offset
+    description: Defines the top offset for sticky headers. This setting value must include the CSS unit type, for example `10px` or `2rem`.
+    var: $theme-table-sticky-top-offset
+    default: '-1px'
+    type: CSS length
```

Comment:
Should this type be `units`? It mentions in the description that the value must include the CSS unit type and would match the `$theme-focus-offset` setting type. 

It looks like this was mentioned in PR for the change but I'm not sure if this was answered elsewhere.

> does this setting accept units too or only pixels? Do we have any guidance for this?

https://github.com/uswds/uswds/pull/5420/files#r1475087073

Reply:
Right now, it isn't run through the units function because it might need precise, non-unit values. It is just a direct definition, so users need to submit what they would add in the Sass for the CSS `top` property. 

Wondering if not accommodating units is the right choice 🤔

Reply:
In that case, I think this is the right decision for now. It's probably important to allow precise control for users since there could be a number of reasons they'll need to offset the header.